sites/partners/src/applications/PaperApplicationForm/sections/FormPrograms.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useMemo } from "react"
```

Comment:
There's a couple variables in here that are named with `preference` that don't look intentional, can you check and change to program?